source/extensions/access_loggers/grpc/grpc_access_log_utils.cc

```diff
@@ -37,7 +37,7 @@ void Utility::responseFlagsToAccessLogResponseFlags(
     envoy::data::accesslog::v3::AccessLogCommon& common_access_log,
     const StreamInfo::StreamInfo& stream_info) {
 
-  static_assert(StreamInfo::ResponseFlag::LastFlag == 0x100000,
+  static_assert(StreamInfo::ResponseFlag::LastFlag == 0x200000,
```

Comment:
You need to change the proto and actually fix the code. ;)

Reply:
Is it ok to change the API surface because of this?

Reply:
Yeah it's just a field addition so backwards compatible.

Reply:
(Maybe also add a comment here for the next person.)

Reply:
Done.